oasislmf/computation/hooks/post_analysis.py

```diff
@@ -1,16 +1,43 @@
-# TO DO -- placeholder for post analysis output modification hook
-
-# __all__ = [
-#    '_'
-# ]
-#
-#
-# class _(ComputationStep):
-#    """
-#    - desc here -
-#    """
-#    step_params = [  .. args ..  ]
-#
-#
-#    def run(self):
-#        return None
+__all__ = [
+    'PostAnalysis'
+]
+
+import os
+
+from ..base import ComputationStep
+from ...utils.path import get_custom_module
+from ...utils.exceptions import OasisException
+
+
+class PostAnalysis(ComputationStep):
+    """Computation step that is called after loss calculations.
+
+    It passes the output directory to a customisable function that might modify or add to the
+    standard output files.
+    """
+    step_params = [
+        {'name': 'post_analysis_module', 'required': True, 'is_path': True, 'pre_exist': True,
+         'help': 'Post-Analysis module path'},
+        {'name': 'post_analysis_class_name', 'default': 'PostAnalysis',
+         'help': 'Name of the class to use for the post_analysis'},
+        {'name': 'model_run_dir', 'is_path': True, 'pre_exist': False,
+         'help': 'Model run directory path'},
+        {'name': 'analysis_settings_json', 'is_path': True, 'pre_exist': True,
+         'help': 'Analysis settings JSON file path'},
+    ]
```

Comment:
Add the `model_data_dir` to future proof this. If there anything else we or others may need in a postprocessing function?